client/src/legacy/GridField.js

```diff
@@ -176,7 +177,8 @@ $.entwine('ss', function($) {
       this.find('.sortable-header').append('<th class="main col-Actions" />');
       this.find('tbody tr').each(function () {
         var cell = $(this).find('td:last');
-        cell.attr('colspan', 2);
+        var colspan = cell.attr('colspan') ?? 1;
```

Comment:
```suggestion
        // Note we need to add 1 to the current column span, because we're going to add a new column
        var colspan = cell.attr('colspan') ?? 1;
```